nvim/init.lua

```diff
@@ -80,6 +80,10 @@ opt('w', 'number', true)              -- print line number
 opt('w', 'wrap', false)               -- disable line wrap
 opt('o', 'updatetime', 100)           -- update frequency
 
+-- onedark.vim override:
+-- don't set a background color just use the terminal's background color
+vim.api.nvim_command("autocmd ColorScheme *  call onedark#set_highlight(\"Normal\", {})")
```

Comment:
this has been a constant source of frusteration with my configuration. I want to look into writing my own one-dark color theme to better understand the issue. It may be that the base theme is trying to be too smart and cover too many cases. For me I only intend to run vim in a terminal which I think makes things simpler around background color.